config/initializers/qualifications/a_and_as_level.rb

```diff
@@ -0,0 +1,187 @@
+A_AND_AS_LEVEL_SUBJECTS = [
+  'Accounting',
+  'Ancient History',
+  'Anthropology',
+  'Applied Art and Design',
+  'Applied Art and Design (Double Award)',
```

Comment:
We should probably change all instances of `Double Award` to `Double award` to be consistent with how we capitalise this in the Science GCSE section

Reply:
Done